src/rdkafka_mock.c

```diff
@@ -41,7 +41,11 @@
 #include "rdkafka_mock.h"
 #include <stdarg.h>
 
+typedef struct rd_kafka_mock_request_s rd_kafka_mock_request_t;
+
 static void rd_kafka_mock_cluster_destroy0(rd_kafka_mock_cluster_t *mcluster);
+static rd_kafka_mock_request_t *
+rd_kafka_mock_request_new(int32_t id, int16_t api_key, rd_ts_t timestamp);
```

Comment:
Instead of rd_ts_t pass int64_t as argument and convert it internally as in future this might need to be exposed publicly and even if not the casting can be done internally in the function rather than externally.

Reply:
Done, addressed